pkcs11/src/new/functions/object_management.rs

```diff
@@ -1,29 +1,29 @@
 use crate::new::types::function::{Result, Rv};
-use crate::new::types::session::Session;
-use crate::new::types::object::Object;
 use crate::new::types::object::Attribute;
+use crate::new::types::object::Object;
+use crate::new::types::session::Session;
 use crate::new::Pkcs11;
 // TODO: get_attribute_value
 
 impl Pkcs11 {
-    pub fn find_objects(&self, session: &mut Session, template: Option<Vec<Attribute>>) -> Result<Vec<Object>> {
+    pub fn find_objects(
+        &self,
+        _session: &mut Session,
+        _template: Option<&[Attribute]>,
+    ) -> Result<Vec<Object>> {
         Ok(Vec::new())
     }
 
-    pub fn create_object(&self, session: &Session, template: Vec<Attribute>) -> Result<Object> {
+    pub fn create_object(&self, _session: &Session, _template: &[Attribute]) -> Result<Object> {
```

Comment:
```suggestion
    pub fn create_object(&self, _session: &Session, _template: &mut [Attribute]) -> Result<Object> {
```

Reply:
Or does this actually work without `mut`?

Reply:
hmm yes you are right, it will need `mut`. Will add it for next PR

Reply:
Meh, whenever we get to implementing that function